### PR TITLE
Initial module setup

### PR DESCRIPTION
**Pull request by KevinJCross**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)
```
Comment (timja) [inline .github/PULL_REQUEST_TEMPLATE.md]: want to grab the other PR template? from most of the repos

File: README.md [hunk 2]
```diff
@@ -1,2 +1,26 @@
 # flexvolume-nodejs
-Azure flex-volume to nodejs property integration.
+
+This module to to incorporate the integration of the Azure keyvault flex volume to node properties.
```
Comment (timja) [inline README.md]:
```suggestion
This module is to incorporate the integration of the Azure keyvault flex volume to node properties.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,26 @@
...
+
+[![Build Status](https://travis-ci.org/hmcts/flexvolume-nodejs.svg?branch=master)](https://travis-ci.org/hmcts/flexvolume-nodejs.svg?branch=master)
```
Comment (timja) [inline README.md]:
```suggestion
[![Build Status](https://travis-ci.com/hmcts/flexvolume-nodejs.svg?branch=master)](https://travis-ci.com/hmcts/flexvolume-nodejs.svg?branch=master)  
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,26 @@
 # flexvolume-nodejs
-Azure flex-volume to nodejs property integration.
+
+This module to to incorporate the integration of the Azure keyvault flex volume to node properties.
+
+[![Build Status](https://travis-ci.org/hmcts/flexvolume-nodejs.svg?branch=master)](https://travis-ci.org/hmcts/flexvolume-nodejs.svg?branch=master)  
+[![MIT license](http://img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)  
+[![Greenkeeper badge](https://badges.greenkeeper.io/hmcts/flexvolume-nodejs.svg)](https://greenkeeper.io/)
```
Comment (timja) [inline README.md]: prefer dependabot over greenkeeper, it bumps the yarn.lock over well

File: __tests__/flexVaultTests.ts [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import { Properties } from "../src";
+
+describe("Asure flex vault module should start up", () => {
```
Comment (timja) [inline __tests__/flexVaultTests.ts]:
```suggestion
describe("Azure flex vault module should start up", () => {
```

File: types/@hmcts/nodejs-logging/Logger.ts [hunk 6]
```diff
@@ -0,0 +1,7 @@
+declare module '@hmcts/nodejs-logging' {
```
Comment (timja) [inline types/@hmcts/nodejs-logging/Logger.ts]: can this be moved to nodejs-logging instead?

File: types/steps.d.ts [hunk 7]
```diff
@@ -0,0 +1,102 @@
+type ICodeceptCallback = (i: CodeceptJS.I, ...steps: any) => void;
```
Comment (timja) [inline types/steps.d.ts]: looks unused?

File: types/steps-override.d.ts [hunk 8]
```diff
@@ -0,0 +1,28 @@
+interface Feature {
```
Comment (timja) [inline types/steps-override.d.ts]: unused?

File: types/expectations.d.ts [hunk 9]
```diff
@@ -0,0 +1,17 @@
+declare namespace Chai {
```
Comment (timja) [inline types/expectations.d.ts]: unused?

File: LICENCE.md [hunk 10]
```diff
@@ -0,0 +1,7 @@
+Copyright 2018 HMCTS
```
Comment (timja) [inline LICENCE.md]: 2019

File: types/@hmcts/nodejs-logging/Logger.ts (hunk 6 above)
Comment (KevinJCross) [reply types/@hmcts/nodejs-logging/Logger.ts]: yup ... i was working on it to get ti to pass first then Ill move it over :D

File: types/steps.d.ts (hunk 7 above)
Comment (KevinJCross) [reply types/steps.d.ts]: removed

File: types/steps-override.d.ts (hunk 8 above)
Comment (KevinJCross) [reply types/steps-override.d.ts]: removed

File: types/expectations.d.ts (hunk 9 above)
Comment (KevinJCross) [reply types/expectations.d.ts]: hmm might need it later but we are using jest in the module so ill remove it

File: LICENCE.md (hunk 10 above)
Comment (KevinJCross) [reply LICENCE.md]: sure

File: README.md [hunk 11]
```diff
@@ -1,2 +1,32 @@
-# flexvolume-nodejs
-Azure flex-volume to nodejs property integration.
+# properties-volume-nodejs
+[![MIT license](http://img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
+<br>[![Greenkeeper badge](https://badges.greenkeeper.io/hmcts/properties-volume-nodejs.svg)](https://greenkeeper.io/)
```
Comment (timja) [inline README.md]: remove please, we use dependabot now instead

File: types/expectations.d.ts (hunk 9 above)
Comment (timja) [reply types/expectations.d.ts]: lets remove this

Comment (KevinJCross) [reply types/expectations.d.ts]: done

File: README.md (hunk 11 above)
Comment (KevinJCross) [reply README.md]: ok np.

**Pull request merged**